lib/src/native/realm_core.dart

```diff
@@ -442,6 +442,10 @@ class _RealmCore {
   void listClear(RealmList list) {
     _realmLib.invokeGetBool(() => _realmLib.realm_list_clear(list.handle._pointer));
   }
+
+  bool equals<T extends NativeType>(Handle<T> x, Handle<T> y) {
```

Comment:
```suggestion
  bool equals(RealmObject first, RealmObject second) {
```

Reply:
Fixed

Reply:
I think we don't need the type constraint on this method.  Added a suggestion